.github/workflows/pydpf-post.yml

```diff
@@ -73,18 +73,48 @@ jobs:
         with:
           python-version: "3.8"
 
-      - name: "Build Core Package"
-        uses: ansys/pydpf-actions/build_package@v2.3
+      - name: "Install ansys-dpf-core build requirements"
+        run: pip install -r requirements/requirements_build.txt
+
+      - name: "Build ansys-dpf-core wheel"
+        shell: bash
+        run: |
+          if [ ${{ matrix.os }} == "ubuntu-latest" ]; then
```

Comment:
is there any place where we have all wheels available (link in pypi) and we let pip pick the right one?

Reply:
@cbellot000 this is not tested in the CI, no.
We know that pip will pick from most-specific to more-generic, I tested that for other libraries which also do things this way.

Reply:
maybe we could at least run the example like that, then (at least in the release pipeline)? if not, we are not really testing a user environment